ztag/transforms/oracle.py

```diff
@@ -29,10 +28,11 @@ def _transform_object(self, obj):
 
         # Otherwise, just copy everything from handshake into the root.
         for k, v in results["handshake"].items():
-            if k == "refuse_error_raw":
-                v = self.clean_banner(v)
-            elif k == "refuse_error":
-                v = self.clean_descriptor(v)
-            zout.transformed[k] = v
+            if v is not None:
```

Comment:
**Nice to have** _([pull request commenting with a purpose](https://gist.github.com/andrewsardone/3fb9746e415d59784e97))_

Could save one indentation level by doing `if v is None: continue`